src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs

```diff
@@ -102,6 +104,148 @@ public TransceiverDirection TransceiverDirection
             }
         }
 
+        /// <summary>
+        /// Gets or sets the transceiver codec of current media source.
+        /// </summary>
+        /// <remarks>
+        /// This API is not supported in <see cref="MediaFileSource"/>, <see cref="MediaPacketSource"/>.<br/>
+        /// The WebRTC must be in the <see cref="WebRTCState.Idle"/> state when transceiver codec is set.
+        /// </remarks>
+        /// <value>A <see cref="TransceiverCodec"/> that specifies the transceiver codec.</value>
+        /// <exception cref="InvalidOperationException">
+        ///     MediaSource is not attached yet.<br/>
+        /// -or-<br/>
+        ///     This MediaSource is not supported type of MediaSource.<br/>
+        /// -or-<br/>
+        /// The WebRTC is not in the valid state.
+        /// </exception>
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <since_tizen> 10 </since_tizen>
+        public TransceiverCodec TransceiverCodec
+        {
+            get
+            {
+                if (!SourceId.HasValue)
+                {
+                    throw new InvalidOperationException("MediaSource is not attached yet. Call AddSource() first.");
+                }
+                if (this is MediaFileSource || this is MediaPacketSource)
+                {
+                    throw new InvalidOperationException($"This property is not supported in {this.GetType()}.");
+                }
+
+                NativeWebRTC.GetTransceiverCodec(WebRtc.Handle, SourceId.Value, MediaType, out TransceiverCodec codec).
+                    ThrowIfFailed("Failed to get transceiver codec");
+
+                return codec;
+            }
+            set
+            {
+                if (!SourceId.HasValue)
+                {
+                    throw new InvalidOperationException("MediaSource is not attached yet. Call AddSource() first.");
+                }
+                if (this is MediaFileSource || this is MediaPacketSource)
+                {
+                    throw new InvalidOperationException($"This property is not supported in {this.GetType()}.");
+                }
+
+                WebRtc.ValidateWebRTCState(WebRTCState.Idle);
+
+                NativeWebRTC.SetTransceiverCodec(WebRtc.Handle, SourceId.Value, MediaType, value).
+                    ThrowIfFailed("Failed to set transceiver codec");
+            }
+        }
+
+        /// <summary>
+        /// Retrieves the current available transceiver codecs.
```

Comment:
IMO "current available" -> "supported" or "available" would be better

Reply:
Changed to `supported`.